tsl/test/t/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(PROVE_TEST_FILES
+    001_simple_multinode.pl
+)
+
+foreach(P_FILE ${PROVE_TEST_FILES})
+    configure_file(${P_FILE} ${CMAKE_CURRENT_BINARY_DIR}/${P_FILE} COPYONLY)
+endforeach(P_FILE)
```

Comment:
Why do you need to copy the files?

Reply:
The TESTDIR variable is used to find the tap tests. It also creates the `tmp_check` subdirectory which contains all the log files for the various nodes that are created. We obviously want it to be part of the `build/tsl/test` location. Hence need to copy the `t` subdirectory pl files into this location.